script/WireAdapters.s.sol

```diff
@@ -65,19 +66,25 @@ contract WireAdapters is Script {
 
             // Wire WormholeAdapter
             if (wormholeAddr != address(0)) {
-                uint16 wormholeId2 = uint16(vm.parseJsonUint(config2, "$.adapters.wormhole.chain-id"));
+                address wormholeAddr2 = vm.parseJsonAddress(config2, "$.contracts.wormholeAdapter");
+                uint16 wormholeId2 = uint16(vm.parseJsonUint(config2, "$.adapters.wormhole.wormholeId"));
+
                 WormholeAdapter wormholeAdapter = WormholeAdapter(wormholeAddr);
-                wormholeAdapter.file("sources", wormholeId2, centrifugeId2, wormholeAddr);
-                wormholeAdapter.file("destinations", centrifugeId2, wormholeId2, wormholeAddr);
+                wormholeAdapter.file("sources", centrifugeId2, wormholeId2, wormholeAddr2);
```

Comment:
Main issue was that the last param here should be of the remote chain, not local chain